src/mustache_core.js

```diff
@@ -206,7 +206,9 @@ var core = {
 					}
 				}
 				// Look up partials in options first.
-				var partial = options.peek("partials." + localPartialName), renderer;
+				var partial = options.peek("partials." + localPartialName);
+				partial = partial || ( options.inlinePartials && options.inlinePartials[ localPartialName ] );
```

Comment:
Good question, I could see either way. This was intentional - Justin’s idea - to potentially allow end user of a component to overwrite them.
@justinbmeyer What do you think; Should we add a dev warning for that before we merge it in?